app/serializers/api/product_serializer.rb

```diff
@@ -49,10 +49,17 @@ class Api::CachedProductSerializer < ActiveModel::Serializer
   has_many :images, serializer: Api::ImageSerializer
   has_one :supplier, serializer: Api::IdSerializer
 
+  #return an unformatted descripton
   def description
     strip_tags object.description
   end
 
+  #return a sanitized html description
+  def description_html
+    d = sanitize(object.description, options = {tags: "p, b, strong, em, i"})
```

Comment:
@oeoeaio I agree in principle, but from the standpoint of malicious POST requests there isn't really any difference between this field and every other field in the whole app (at least all the textarea fields), whether they have textAngular or not. Last time I looked I couldn't see any pre-save validation for any fields in any of the models, so I sort of feel like if this field isn't secure enough then everything needs a security review. Or am I missing something?